Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -890,11 +891,37 @@ SWIFT_CLASS("_TtC10RevenueCat15ProductsManager")
 /// </ul>
 SWIFT_CLASS_NAMED("PromotionalOffer")
 @interface RCPromotionalOffer : NSObject
+/// The <code>StoreProductDiscount</code> in this offer.
+@property (nonatomic, readonly, strong) RCStoreProductDiscount * _Nonnull discount;
+/// The <code>SignedData-swift.class</code> provides information about the <code>PromotionalOffer</code>’s signature.
+@property (nonatomic, readonly, strong) RCPromotionalOfferSignedData * _Nonnull signedData;
 - (nonnull instancetype)init SWIFT_UNAVAILABLE;
 + (nonnull instancetype)new SWIFT_UNAVAILABLE_MSG("-init is unavailable");
 @end
 
 
+@interface RCPromotionalOffer (SWIFT_EXTENSION(RevenueCat))
+@end
+
+@class NSUUID;
+
+/// Contains the details of a promotional offer discount that you want to apply to a payment.
+SWIFT_CLASS_NAMED("SignedData")
```

Comment:
I think I'm not going to mention #1523 in the release notes since users don't really need it.